openstep_parser/openstep_parser.py

```diff
@@ -125,51 +130,67 @@ def _parse_array_entry(self, str, index, array):
         return index + 1
 
     def _parse_padding(self, str, index):
-        index = self._ignore_whitespaces(str, index)
-        index = self._ignore_comment(str, index)
-        index = self._ignore_whitespaces(str, index)
+        str_len = len(str)
```

Comment:
does the inlining really improve that much the performance?

Reply:
On my example project it goes from ~2.3 to ~2.0 seconds just due to this inlining, but yeah when doing ~1.4 million calls of `_parse_padding` the function call overhead adds up 😕 

There's an even more aggressive option I've tried that saves even more time (similarly by avoiding function calls) which is to replace `_parse_padding` with
```python
_PADDING_RE = re.compile(r'\s*(?:/\*.*?\*/)?\s*')

...

index = _PADDING_RE.match(str, index).end()
```
but that's *definitely* a bit harder to read and more invasive of a change (though it does get the parsing for this example project down to ~1.7 seconds).